system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# description: ManageIQ appliance console launcher
+#
+
+if [ -d /var/www/miq/vmdb ] ; then
+  cd /var/www/miq/vmdb
+else
+  cd $(dirname $0)/../../../vmdb
```

Comment:
I don't understand why it's handling non-appliance setups here or what it's doing.  Can you explain that?